src/toil/leader.py

```diff
@@ -182,49 +527,81 @@ def issueJobs(self, jobs):
         """
         for job in jobs:
             self.issueJob(job)
+            
+    def issueServiceJob(self, jobNode):
+        """
+        Issue a service job, putting it on a queue if the maximum number of service
+        jobs to be scheduled has been reached.
+        """
+        if jobNode.preemptable:
+            self.preemptableServiceJobsToBeIssued.append(jobNode)
+        else:
+            self.serviceJobsToBeIssued.append(jobNode)
+        self.issueQueingServiceJobs()
+    
+    def issueQueingServiceJobs(self):
```

Comment:
Wouldn't this make more sense in the service manager? It took me a while to find this code